Automattic/jetpack
# Admin UI: Adding h1 page headings for screen reader navigation

This partially addresses issues mentioned in #6533

#### Changes proposed in this Pull Request:

* In this PR, h1 headings have been added to the following admin pages:
  * `wp-admin/admin.php?page=jetpack#/dashboard`
  * `wp-admin/admin.php?page=jetpack#/my-plan`
  * `wp-admin/admin.php?page=jetpack#/recommendations/summary`
  * `wp-admin/admin.php?page=jetpack#/security`
  * `wp-admin/admin.php?page=jetpack#/performance`
  * `wp-admin/admin.php?page=jetpack#/writing`
  * `wp-admin/admin.php?page=jetpack#/sharing`
  * `wp-admin/admin.php?page=jetpack#/discussion`
  * `wp-admin/admin.php?page=jetpack#/traffic`
  * `wp-admin/admin.php?page=stats`
  * `wp-admin/admin.php?page=jetpack#/privacy`
  * `wp-admin/admin.php?page=jetpack_about`
  * `wp-admin/admin.php?page=jetpack_modules`

#### Other information:



#### Jetpack Product Discussion

n/a.

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Visit the listed pages with this PR checked out. You shouldn't notice any difference visually.
* Check different screen-sizes - there should still be no difference.
* Inspecting the source code, you should see an h1 tag added with an appropriate heading:


<img width="753" alt="Screenshot 2022-07-04 at 09 00 44" src="https://user-images.githubusercontent.com/16754605/177109971-57d8ac0b-402c-401e-aa81-37e8a4b2bc12.png">

* Design review shouldn't be needed as these elements are being hidden (except on the Site Stats page, where the h2 has been changed to a h1) - using the WordPress [screen-reader-text class ](https://make.wordpress.org/accessibility/handbook/markup/the-css-class-screen-reader-text/)(thank you @bluefuton for mentioning this)
